frontend/packages/shared/src/enums/LibraryContentType.ts

```diff
@@ -0,0 +1,4 @@
+export enum LibraryContentType {
+  CodeList = 'CodeList',
+  TextResource = 'TextResource',
```

Comment:
The endpoint will accept camelCase if you want to use that 🙂 
```suggestion
  CodeList = 'codeList',
  TextResource = 'textResource',
```